package.json

```diff
@@ -44,7 +44,7 @@
     "@prantlf/jsonlint": "^14.0.3",
     "@uswds/uswds": "^3.9.0",
     "autoprefixer": "^10.4.16",
-    "del": "^6.1.1",
+    "concurrently": "^9.1.0",
     "dotenv": "^16.3.1",
     "eslint": "^8.56.0",
```

Comment:
@RileySeaburg We should update to `v9` during the next POAM. Made a note of this.